### DISCUSSION TITLE
Please return the "free" days for to-do list.

### DISCUSSION THREAD

**Discussion by Backha** [xanthics/genshin_grind_planner#5] [General]
Weekdays that you have nothing to farm were very useful if I wanted to see if I can just farm laylines.

**Comment by xanthics** [comment]
Could you please elaborate?  I haven't changed anything with how the farming list is displayed in weeks.

![image](https://user-images.githubusercontent.com/3137623/112533877-a7206c80-8d67-11eb-9bc6-3495f8aa6a30.png)

**Reply by xanthics** [reply]
Poking around for 2 minutes and i see what is going on.  It only shows the blank days like that if you have something to farm on that day but already have enough.  Otherwise the day doesn't show.  I'll push a tweak to always show each day soonish.

**Reply by xanthics** [reply]
Should be addressed with https://github.com/xanthics/genshin_grind_planner/commit/eabac61d720617e1a726479e1abdf44c1772396a as it now explicitly says there is nothing to do on a day.

![image](https://user-images.githubusercontent.com/3137623/112551004-79deb900-8d7d-11eb-9c78-c2fbf83fd926.png)
